checkstyle/suppressions.xml

```diff
@@ -56,6 +56,11 @@
     <suppress checks="MethodCount" files="com/hazelcast/cache/impl/CacheStatisticsImpl"/>
     <suppress checks="ClassDataAbstractionCoupling" files="com/hazelcast/cache/impl/DefaultOperationProvider"/>
 
+    <!-- Cardinality -->
+    <suppress checks="JavadocMethod" files="com/hazelcast/cardinality/"/>
```

Comment:
I believe that currently there was a non-formal decision to create new suppressions as annotations and not add them in the checkstyle xml. The problem with package-wide suppressions is that they automatically suppress warnings on new classes. David Sondermann is the authority on code analysis so we can contact him and reach a common agreement.